Week1/homework/js-exercises/logHello.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+var a = 'Hello'+ " " +'world!';//english
```

Comment:
I would remove the `var a` here and just directly log the value. Like this:

`console.log('Hello'+ " " +'world!');` 

And also for all the others:)